install-app.sh

```
@@ -10,11 +10,11 @@
 apt-get update -yq
 apt-get install python3-pip -yq
 
-# Create a directory for the app and download the files. 
-mkdir /app 
+# Create a directory for the app and download the files.
+mkdir /app
```

The script creates /app but does not ensure git is installed. The next statements clone a repo, so consider installing git (e.g. `apt-get install git -yq`) before running `git clone` to avoid runtime failure. Also consider adding `set -e` at the top so the script fails fast and the VM extension logs the error.